manifests/net_process_log.pp

```diff
@@ -0,0 +1,60 @@
+# @summary Logs user processes and open network connections to syslog
+#
+# Logs user processes and open network connections to syslog
+#
+# @param ignore_users
+#   String to pass to net_process_log.pl script as -i argument.
+#   Setting to empty string will use default defined in net_process_log.pl
```

Comment:
Can you supply examples of what a list of ignored users would look like?

Reply:
Added an example list of users and gave the format expected.